docs/changes/coordinates/12556.bugfix.rst

```diff
@@ -0,0 +1,6 @@
+Fixed a bug where instantiating a coordinate representation without copying the
```

Comment:
@saimn , is this too long? I vaguely remember some discussion on the length of change log entry before but I don't remember the conclusion.

Reply:
Yeah, I went way technical.  How about only mentioning the most likely way a user would encounter this issue:
```
Fixed a bug where changing the wrap angle of the longitude component of
a representation could raise a warning or error in certain situations.
```

Reply:
Not sure if we agreed on something. IMO, shorter is better though sometimes it may be useful to give more details.

Reply:
Replaced with the shorter version